style.css

```diff
@@ -100,6 +100,186 @@
   text-align: center;
 }
 
+.footer {
+  margin-top: 42px;
+  background: #fff url(/images/bg-cover-page.jpg) no-repeat;
+  background-size: cover;
+  background-position: center;
+  width: auto;
+  height: 250px;
+}
+
+/* Work-section */
+.work {
+  display: grid;
+  grid-template-columns: 400px;
+}
+
+.work h1 {
+  font-family: 'Playfair', sans-serif;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 50px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.work .project {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: fit-content;
+  background: #1a2236;
+  justify-content: flex-start;
+  color: #fff;
+  padding: 0;
+}
+
+.project img {
+  width: 100%;
+  background-clip: border-box;
+}
+
+.project h3 {
+  margin: 25px 32px 0 40px;
+  font-family: 'Poppins', 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-size: 26px;
+  font-weight: lighter;
+  line-height: 16px;
+  letter-spacing: 0.05em;
+  text-align: left;
+}
+
+.project .list-tech {
+  list-style: none;
+  margin: 20px 20px 0 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  align-items: flex-start;
+}
```

Comment:
- [ ] [OPTIONAL] Please make sure the list of tech stacks is laid out properly. The Figma design is a bit different from your output.